src/api/assets/assetsApi.ts

```diff
@@ -86,12 +86,28 @@ export const assetsQueries = createQueriesCollection({
   },
   // TODO: This may be moved in a balancesApi file in the future?
   balances: {
-    addressToken: (addressHash: string, tokenId: string) => ({
-      queryKey: ['addressTokenBalance', addressHash, tokenId],
-      queryFn: () =>
-        client.explorer.addresses
-          .getAddressesAddressTokensTokenIdBalance(addressHash, tokenId)
-          .then((balance) => ({ ...balance, id: tokenId }))
+    addressTokens: (addressHash: string, tokenIds: string[]) => ({
+      queryKey: ['addressTokensBalance', addressHash, tokenIds.toString()],
```

Comment:
why is `tokenIds` needed since they are not an input to the endpoint?

Reply:
Haha, I was so sure you'd need to pass those that I didn't even double checked 🤦 
Much better than what I thought. Updating..